pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
-name = "project-template"
+name = "ConfiGator"
 version = "0.1.0"
-description = "Simple Python project template"
-authors = [""]
+description = "A Configuration File Generator for GatorGradle"
+authors = ["Kyrie Doniz", "Kevin Lee", "Kai'lani Woodard"]
```

Comment:
The full list of authors should be added.

Reply:
Please address this issue for this release.